examples/api/test/widgets/editable_text/editable_text.on_changed.0_test.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/editable_text/editable_text.on_changed.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Does not show dialog when answer is not correct', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.OnChangedExampleApp(),
+    );
+
```

Comment:
Should there be a check here to confirm that 'What number comes next in the sequence?' and '1, 1, 2, 3, 5, 8...?', are being displayed?

Reply:
@victorsanni thanks for the review and suggestions. It makes sense to add such a test!
I did this in the latest commit.